<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgPachy</class>
 <widget class="QWidget" name="WidgPachy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>111</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>111</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>272</width>
    <height>111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="TonoGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>150</width>
     <height>95</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>95</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>95</height>
    </size>
   </property>
   <property name="title">
    <string>Pachymétrie</string>
   </property>
   <widget class="UpSpinBox" name="PachyOGSpinBox">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>45</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="suffix">
     <string>µ</string>
    </property>
    <property name="minimum">
     <number>350</number>
    </property>
    <property name="maximum">
     <number>700</number>
    </property>
    <property name="value">
     <number>549</number>
    </property>
   </widget>
   <widget class="UpSpinBox" name="PachyODSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="suffix">
     <string>µ</string>
    </property>
    <property name="minimum">
     <number>350</number>
    </property>
    <property name="maximum">
     <number>700</number>
    </property>
    <property name="value">
     <number>549</number>
    </property>
   </widget>
   <widget class="QLabel" name="OGlabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OG</string>
    </property>
   </widget>
   <widget class="QLabel" name="ODlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OD</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="MethodegroupBox">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>7</y>
     <width>120</width>
     <height>95</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>95</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>95</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>95</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="EchoRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Echo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="OptiqueRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Optique</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="OCTRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>69</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>OCT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpSpinBox</class>
   <extends>QSpinBox</extends>
   <header>upspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
